.github/workflows/ci.yaml

```diff
@@ -398,9 +398,9 @@ jobs:
       - name: Create Kubernetes cluster
         id: kind
         if: ${{ steps.filter-images.outputs.kube-webhook-certgen == 'true' }}
-        uses: engineerd/setup-kind@aa272fe2a7309878ffc2a81c56cfe3ef108ae7d0 # v0.5.0
+        uses: engineerd/setup-kind@v0.5.0
         with:
-          version: v0.11.1
+          version: v0.14.0
           image: kindest/node:v1.21.1
```

Comment:
could you bump it to `kindest/node:v1.21.12` ? https://github.com/kubernetes-sigs/kind/releases/tag/v0.14.0

Reply:
bumped node image to 1.21.12